envVariables.js

```diff
@@ -85,6 +85,10 @@ let enviromentVariables = {
   "DISABLE_LEARNER_SERVICE_ON_OFF": {
     "message" : "Disable learner service on/off",
     "default": "ON"
+  },
+  "SUNBIRD_SERVICE_URL" : {
```

Comment:
@VISHNUDAS-tunerlabs add this in .env.sample also

Reply:
not using SUNBIRD_SERVICE_URL. so removed it. adding to .env.sample not required